include/boost/beast/core/async_base.hpp

```diff
@@ -460,7 +460,7 @@ class async_base
     @code
 
     // Asynchronously send a message multiple times, once per second
-    template <class AsyncWriteStream, class T, class WriteHandler>
+    template <class AsyncWriteStream, class T, BOOST_BEAST_ASYNC_TPARAM2 WriteHandler>
```

Comment:
This is expository (i.e. in the docs) do we want the docs to show this macro?

Reply:
If we don't, how will users write async completion handlers?
Should we perhaps spell out the different approaches for when concepts are enabled and not?